lib/Targets/AIETargets.cpp

```diff
@@ -398,17 +398,13 @@ SECTIONS
               }
             };
 
-            if (auto tile = target_model.getMemSouth(srcCoord))
-              doBuffer(tile, target_model.getMemSouthBaseAddress(),
+            doBuffer(target_model.getMemSouth(srcCoord), target_model.getMemSouthBaseAddress(),
```

Comment:
Hmm... probably the ld script generation should have similar structure?

Reply:
Yeah looks like it. I'll add the change to this PR